packages/widget_toolkit/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [0.1.1]
+ - Fixed loading state's loading indicator on `SmallButton` widget
```

Comment:
We can simplify this to "Fixed loading state of 'SmallButton' widget"

Reply:
Done